apps/webpage/src/app/archive/archive.component.ts

```diff
@@ -0,0 +1,222 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+interface Talk {
+  title: string;
+  author: string;
+  slidesUrl?: string;
+  description?: string;
+}
+
+export interface Meetup {
+  date: string;
+  location: string;
+  talks?: Talk[];
+}
+
+@Component({
+  selector: 'ng-stuttgart-archive',
+  templateUrl: './archive.component.html',
+  styleUrls: ['./archive.component.scss']
+})
+export class ArchiveComponent implements OnInit {
+  meetupIndex = 0;
+  selectedMeetup$: Observable<Meetup>;
+  meetups$: Observable<Meetup[]>;
+
+  selectMeetup(index: number) {
+    this.meetupIndex = index;
+    this.selectedMeetup$ = this.meetups$.pipe(
```

Comment:
OnPush ChangeDetection not possible because of this replacement of class field selectedMeetup$

Reply:
I see what you mean now, you're right. I tried to solve it deborah kurata style (see commented out code), but it doesn't work yet. help needed.